{"name": "test_add_member_fail[user2-11528872802-手机已存在于通讯录，无法添加]", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'implicitly_wait'", "trace": "self = <testcases.test_addMember.TestAddMember object at 0x000001FE846DDBA8>\n\n    def setup(self):\n>       self.app = App()\n\ntestcases\\test_addMember.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <page.app.App object at 0x000001FE846DDD68>, driver = None\n\n    def __init__(self, driver: WebDriver = None):\n        self._driver = driver\n>       self._driver.implicitly_wait(5)\nE       AttributeError: 'NoneType' object has no attribute 'implicitly_wait'\n\npage\\base_page.py:22: AttributeError"}, "parameters": [{"name": "user", "value": "'user2'"}, {"name": "phone", "value": "'11528872802'"}, {"name": "expect", "value": "'手机已存在于通讯录，无法添加'"}], "start": 1626593578151, "stop": 1626593578151, "uuid": "c36bce1d-e13a-4b38-8b66-530ca9381445", "historyId": "f072b275c56a30d0c23d296fd0a67095", "testCaseId": "b251c483add15ec609094c40b1ea12dc", "fullName": "testcases.test_addMember.TestAddMember#test_add_member_fail", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_addMember"}, {"name": "subSuite", "value": "TestAddMember"}, {"name": "host", "value": "z"}, {"name": "thread", "value": "22104-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_addMember"}]}